app/routes/start-application.js

```diff
@@ -23,22 +29,38 @@ module.exports = [{
   path: '/start-application',
   options: {
     validate: {
-      payload: schema
+      payload: Joi.object()
+        .concat(sbiSchema)
+        .concat(agreementNumberSchema)
```

Comment:
Should there not be a failAction if the schema fails?

Reply:
Similar to the comment below. Not mentioned in the ticket and before working on this, there wasn't a failAction even though the schema was already checking for the SBI. It redirects the user to a 400 page if any validation fails which I think is good enough for now.